tools/lintapidiff.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+TAGS=$(git tag|grep '^[0-9]*.[0-9]*.[0-9]*$'|grep -v '^[12].')
+ocamlfind ocamlopt -package compiler-libs.common,str tools/lintapidiff.ml \
```

Comment:
I see that you documented the assumption that this tool relies on an existing installation with `findlib`, but is there any particular reason for this, or just that you found it easier to iterate on it as a separate tool? I think the reasoning can go both ways: if it's in ` tools` it would be consistent to link with the compiler distribution directly, but having it external like this makes it easier for other people to just steal the scripts and use them for their projects. I like that second advantage and I suspect it could also benefit some other tools, so I would leave things the way you created them.